src/legacy/core_plugins/kibana/public/dashboard/top_nav/add_panel.js

```diff
@@ -159,25 +83,30 @@ class DashboardAddPanelUi extends React.Component {
             </h1>
           </EuiTitle>
 
-          <EuiTabs>
-            {this.renderTabs()}
-          </EuiTabs>
-
           <EuiSpacer size="s" />
 
-          {this.state.selectedTab.savedObjectFinder}
-
+          <SavedObjectFinder
+            onChoose={this.onAddPanel}
+            savedObjectMetaData={this.props.embeddableFactories
+              .filter(embeddableFactory => Boolean(embeddableFactory.savedObjectMetaData))
+              .map(({ savedObjectMetaData }) => savedObjectMetaData)}
+            showFilter={true}
+            callToActionButton={addNewVisBtn}
```

Comment:
This button should live in the Flyout's footer. There're more vertical room to work with. Example:

<img width="555" alt="screen shot 2019-03-07 at 14 06 52 pm" src="https://user-images.githubusercontent.com/549577/53982204-4ff8c200-40e2-11e9-9a53-6da3e8fcde51.png">